src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,57 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.UserInvalidDataException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int MIN_LENGTH = 6;
+    public static final int MIN_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        dataVerification(user);
+        storageDao.add(user);
+        return user;
+    }
+
+    private void dataVerification(User user) {
```

Comment:
Method names are better to be verbs
```suggestion
    private void verifyData(User user) {
```